core/backend/src/ECDb.ts

```diff
@@ -311,6 +311,7 @@ export class ECDb implements IDisposable {
    * @returns Returns the query result as an *AsyncIterableIterator<any>*  which lazy load result as needed. The row format is determined by *rowFormat* parameter.
    * See [ECSQL row format]($docs/learning/ECSQLRowFormat) for details about the format of the returned rows.
    * @throws [IModelError]($common) If there was any error while submitting, preparing or stepping into query
+   * @deprecated in 3.7. Use [[createQueryReader]] instead; it accepts the same parameters.
```

Comment:
`createQueryReader` is beta.

Reply:
Talked to Colin. We are going to bring them out of beta in 4.x.